packages/ui/src/components/labels/CountWithIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import '@ferlab/style/components/labels/CountWithIcon.scss'
+
+export enum CountWithIconTypeEnum  {
+    Inline = 'inline',
+    Stack = 'stack',
+}
+
+export type ICountWithIconProps = {
+    className?: string;
+    Icon: React.ReactNode;
+    type?: CountWithIconTypeEnum;
+    label: React.ReactNode;
+    total: React.ReactNode;
+    style?: React.CSSProperties;
+}
+
+const CountWithIcon = ({
+                           className = '',
+                           Icon,
+                           label,
+                           total,
+                           type = CountWithIconTypeEnum.Stack,
+                           style = {},
+                       }: ICountWithIconProps): React.ReactElement => {
+    return (
```

Comment:
no need for `{ return ...`

Reply:
Maybe we could add something in the eslint to reformat to this when its not needed. (I did it in cqdg-portal)